src/allreduce_robust.cc

```diff
@@ -270,6 +275,9 @@ void AllreduceRobust::CheckPoint_(const Serializable *global_model,
       if (RecoverExec(NULL, 0, 0, ActionSummary::kLocalCheckPoint)) break;
       // save model model to new version place
       int new_version = !local_chkpt_version;
+      printf("[%d] local checkpoint # %d, new # %d \n",
+              this->rank, local_chkpt_version, new_version);
```

Comment:
I think this line does covey anything beyond what Line 307 says,,,we may only need one of them

Reply:
removed